string objectpoint
float32 objx
float32 objy
float32 objz
string imagepoint
float32 imgx
float32 imgy
